README.md

```diff
@@ -49,11 +49,12 @@ Messages sent to the server
 
 Messages sent to the client
 - Receives START :
-  - FAIL : sent if no word is found corresponding to the number of letter given by START
-  - CORRECT [updatedWord] : sends an updated word. Unknown letters are sent as ```_```
+  - FAIL [reason] : sent if no word is found corresponding to the number of letter given by START
```

Comment:
would be best to update the documentation to explain what is the reason sent back to reflect the changes there